requirements-minimal.txt

```diff
@@ -10,6 +10,6 @@ jsonschema>=3.2.0
 psutil>=5.8.0
 lxml>=4.6.5
 spikeextractors>=0.9.7
-spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git
+spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git@master
```

Comment:
.

Reply:
Yeah, this doesn't actually change functionality. Just makes it more explicit (especially to distinguish 'main' (new repo's) vs. 'master' (old repo's).